stories/core/ListingCell.js

```diff
@@ -32,7 +36,7 @@ export const DefaultListingCell = (
       <div>
         <ListingComponents.Bathroom />
         <h1>hello</h1>
-        <ListingComponents.Ratings />
+        <ListingComponents.Ratings {...ratingProps} />
```

Comment:
Why are we changing this to pass props directly to the ratings listing component?
Isn't the point that they just get their props through the `ListingCell`?

Reply:
The idea is to give you the ability to both. So settings like fill color and such can be passed as props. And then the rating score and label will go in via context through `ListingCell`.

Reply:
ok cool 👍